app/controllers/agendapoints/edit.js

```diff
@@ -69,6 +69,41 @@ export default class AgendapointsEditController extends Controller {
     return null;
   }
 
+  get isBusy() {
+    return (
+      !this.editorSetup ||
+      this.saveTask.isRunning ||
+      this.copyAgendapunt.isRunning
+    );
+  }
+
+  get busyText() {
+    if (!this.editorSetup) {
+      return this.intl.t('rdfa-editor-container.loading');
+    }
+    if (this.saveTask.isRunning) {
+      return this.intl.t('rdfa-editor-container.making-copy');
+    }
+    if (this.copyAgendapunt.isRunning) {
+      return this.intl.t('rdfa-editor-container.saving');
+    }
+    return '';
+  }
+
+  setSchemaAndPlugins = modifier(() => {
+    const { schema, plugins } =
+      this.agendapointEditor.getSchemaAndPlugins(false);
+    this.schema = schema;
+    const citationPlugin = this.agendapointEditor.citationPlugin;
```

Comment:
so now this means that anywhere we use this service, we just have to remember to treat the citationPlugin property in this special way?
this doesn't seem like a good solution...
something is off here, we're making a fundamentally simple thing complicated
ember is not being very helpful in all this, but I do have some ideas, I'll have a look